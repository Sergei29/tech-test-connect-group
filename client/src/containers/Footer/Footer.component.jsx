import React from "react";
//style:
import { FooterContainer } from "./Footer.style";

const Footer = ({ children }) => {
  return (
    <FooterContainer>
      {children}
      <small>
        This page is created as a technical test for training purposes only.
      </small>
    </FooterContainer>
  );
};

export default Footer;
