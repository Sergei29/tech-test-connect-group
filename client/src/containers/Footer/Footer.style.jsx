import styled from "styled-components";
import { spacing } from "../../constants/styleVariables";

export const FooterContainer = styled.footer`
  display: flex;
  flex-direction: column;
  text-align: center;
  position: absolute;
  left: 0;
  bottom: 0;
  width: 100%;
  height: ${spacing.footerHeight};
  small {
    font-size: 10px;
  }
`;
