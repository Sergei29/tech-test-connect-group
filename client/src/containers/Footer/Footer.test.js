import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer.component";

describe("Footer", () => {
  let wrapper;

  it("should match the snapshot", () => {
    wrapper = shallow(<Footer></Footer>);
    expect(wrapper).toMatchSnapshot();
  });

  it("should have content displayed if there is any", () => {
    wrapper = shallow(
      <Footer>
        <p>Footer content.</p>
      </Footer>
    );
    expect(wrapper.contains(<p>Footer content.</p>)).toBe(true);
  });
});
