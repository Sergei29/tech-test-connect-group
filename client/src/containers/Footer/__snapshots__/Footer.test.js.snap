// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
<styled.footer>
  <small>
    This page is created as a technical test for training purposes only.
  </small>
</styled.footer>
`;
